helmfile/values/velero.yaml.gotmpl

```diff
@@ -35,7 +35,7 @@ configuration:
     {{- if eq .Values.objectStorage.type "s3" }}
     # Cloud provider where backups should be stored. Usually should
     # match `configuration.provider`. Required.
-    name: aws
+    name: default
```

Comment:
I assume that you need to change gcs to default as well.

Reply:
> Have you checked upgradeability of these changes?
> I think I've only had bad experiences fiddling with storageLocations in velero?
> I mean I suppose the "old" storagelocation "aws/gcs" will still be left. Will velero create the new storageLocation by itself?

Velero created the new storageLocation yes. 
What where you thinking about upgradeability? What i saw this should be fine in the next version and it should be fine reverting it to be called aws/gcs.